### DISCUSSION TITLE
Several issues with Groups

### DISCUSSION THREAD

**Discussion by shazzap** [mbnuqw/sidebery#937] [General]
Adding an existing tab to an existing group or dragging a tab out of an existing group does not update the Group web page.

1. Create a group of a few tabs.
2. Now drag another existing tab into the group (either drag it on to the group tab into its list.
3. Now click the group tab to display the Group page. The Group page only list the tabs that were used to initially create the group. In Sidebery 4.0, the Group Page would automatically be updated to reflect the change of its children list.
4. Same is true if you drag a group's child tab out of the group. The Group page is not updated unless you manually refresh.

Plus Sign to add a new tab within the Group Results in creating a new tag OUTSIDE of the group and once again is not reflected on the group's page.

Flattening a Group leaves a group tab with no children tabs. Also that group tab's group page also still show entries for those (no longer children) tabs

**Comment by mbnuqw** [comment]
The lack of reactivity of the group page is a known issue, will be fixed.

I can't reproduce creating the new tab outside the group, you can open a new bug report for this case.

`Flattening a Group leaves a group tab with no children tabs.`
Hm... indeed, it seems safe to auto-close group tab in this case, thanks for the suggestion.

Moving to discussions due to multiple topics in a single issue.
